src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rental = 40;
+  const longTerm = 7;
+  const shortTerm = 3;
+  const totalPrise = days * rental;
+
+  if (days < shortTerm) {
+    return totalPrise;
+  } else if (days >= shortTerm && days < longTerm) {
```

Comment:
Avoid using 'else' after 'return'. You can remove the 'else' keyword and unindent the 'if' statement since the previous 'if' block contains a 'return' statement. This simplifies the control flow.